app/models/irregular_income_payment.rb

```diff
@@ -0,0 +1,23 @@
+class IrregularIncomePayment < ApplicationRecord
+  ANNUAL_FREQUENCY = 'annual'.freeze
```

Comment:
I'm not sure we need this two-level approach here (of `IrregularIncome` and `IrregularIncomePayment`).  It is needed in some other types precisely because there are multiple payments of one type but here I'm anticipating that there will only be one student loan payment (and one of any other irregular payment types we may come up against in the future.  So I think we just need `GrossIncomeSummary` has many `IrregularIncomePayments` and no `IrregularIncome` in between.